docs/advanced/app-connectors.md

```diff
@@ -30,13 +30,84 @@ If you haven’t already created an app Subgraph yet, please follow the steps in
 
 ### Create the Connector
 
-Coming soon!
+Implementing a connector consists of calling the `createAppConnector()`, which takes care of doing some necessary checks as well as passing contextual information to the callback.
 
-#### Define the queries
+This is how an app connector can get implemented in the simplest way:
 
-#### Call the queries from the connector
+```js
+import { createAppConnector } from '@aragon/connect-core'
 
-#### Define the entities
+export default createAppConnector(() => ({
+  total: () => fetchTotal(),
+}))
+```
+
+From the point of view of a consumer of your connector, the resulting object will be a function that can get used to connect an app.
+
+For example, this is how app authors can initiate a connection to the Voting app. In this example, `connectVoting` got returned by `createAppConnector()`:
+
+```js
+import connect from '@aragon/connect'
+import connectVoting from '@aragon/connect-voting'
+
+// Connect to an organization on mainnet via The Graph.
+const org = await connect('myorg.aragonid.eth', 'thegraph')
+
+// Get and connect to the voting app.
+const voting = await connectVoting(org.app('voting'))
+
+// Fetch the votes from the voting app.
+const votes = await voting.votes()
+```
+
+Following the same example, this is how the connector for the Voting app is implemented:
+
+```js
+import { createAppConnector } from '@aragon/connect-core'
+import Voting from './entities/Voting'
+import VotingConnectorTheGraph, {
+  subgraphUrlFromChainId,
+} from './thegraph/connector'
+
+export default createAppConnector(
+  ({ app, config, connector, network, verbose }) => {
+    if (connector !== 'thegraph') {
+      console.warn(
+        `Connector unsupported: ${connector}. Using "thegraph" instead.`
+      )
+    }
+
+    return new MyAppConnector(
```

Comment:
I fix a small typo here where we should use `Voting` instead of MyAppConnector. Or did you have something different in mind when writing it?